src/components/VencordSettings/ThemesTab.tsx

```diff
@@ -16,15 +16,33 @@
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
 */
 
+import "./styles.css";
+
 import { useSettings } from "@api/settings";
 import ErrorBoundary from "@components/ErrorBoundary";
+import { Flex } from "@components/Flex";
 import { Link } from "@components/Link";
+import { Switch } from "@components/Switch";
+import IpcEvents from "@utils/IpcEvents";
 import { Margins } from "@utils/margins";
 import { useAwaiter } from "@utils/misc";
-import { findLazy } from "@webpack";
-import { Card, Forms, React, TextArea } from "@webpack/common";
+import { findByCodeLazy, findLazy } from "@webpack";
+import { Button, Card, Forms, React, Text, TextArea } from "@webpack/common";
```

Comment:
Not needed but might be nicer to read if you selectively import react types from the types package and import hooks from the webpack common
```suggestion
import { Button, Card, Forms, Text, TextArea, useEffect, useRef, useState } from "@webpack/common";
import { ComponentType, ReactNode, Ref, SyntheticEvent } from "react";
```